packages/wrangler/src/config/patch-config.ts

```diff
@@ -0,0 +1,63 @@
+import TOML from "@iarna/toml";
+import { applyEdits, format, parse as JSONCParse, modify } from "jsonc-parser";
+import { parseTOML, readFileSync } from "../parse";
+import type { RawConfig } from "./config";
+import type { JSONPath } from "jsonc-parser";
+
+export const experimental_patchConfig = (
+	configPath: string | undefined,
+	patch: RawConfig
+) => {
+	if (!configPath) {
+		return;
+	}
+
+	let configString = readFileSync(configPath);
+
+	if (configPath.endsWith("toml")) {
+		// the TOML parser we use does not preserve comments
+		if (configString.includes("#")) {
+			return;
```

Comment:
We need to know if the patch does not happens so we can show a different message to the user?